### ISSUE TITLE
[mesheryctl] `environment create ` command panic error

### ISSUE THREAD

**Issue by Jougan-0** [meshery/meshery#11314]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior

When running `mesheryctl exp environment create --orgId [orgID] -n [name] --description [description]` returns panic.
### Desired Behavior

- [ ] Fix panic error.
- [ ] Update the `orgID` field from `orgId`

### Screenshots/Logs

```
./mesheryctl exp environment create --orgId b42adb3e-26dd-4668-aea7-bf040d872023 --name shlok --description hello 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x3b7258e]

goroutine 1 [running]:
github.com/layer5io/meshery/mesheryctl/internal/cli/root/environments.glob..func3(0xc00050d700?, {0x459a580?, 0x4?, 0x459a584?})
        /home/jougan/OpenSource/meshery/mesheryctl/internal/cli/root/environments/create.go:112 +0x20e
github.com/spf13/cobra.(*Command).execute(0x66419e0, {0xc000981440, 0x6, 0x6})
        /home/jougan/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0xabc
github.com/spf13/cobra.(*Command).ExecuteC(0x66333e0)
        /home/jougan/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        /home/jougan/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/layer5io/meshery/mesheryctl/internal/cli/root.Execute()
        /home/jougan/OpenSource/meshery/mesheryctl/internal/cli/root/root.go:85 +0x45
main.main()
        /home/jougan/OpenSource/meshery/mesheryctl/cmd/mesheryctl/main.go:26 +0x13
```

### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by Redarcher9** [comment]
Hi @Jougan-0,
Can i take this up?

**Comment by Jougan-0** [comment]
Could you confirm that whether you were able to run `mesheryctl` locally?

**Comment by 102vansh** [comment]
can i take up this issue ? I have installed mesheryctl locally

**Comment by Redarcher9** [comment]
Hey @102vansh I'm already working on this.
Sorry @Jougan-0 I did not confirm this back, i was able to set up mesheryctl locally on my machine.

**Comment by Jougan-0** [comment]
@102vansh there are number of open issue ping me under them I'll assign them to you.

**Comment by RRK1000** [comment]
Hi @Jougan-0,
I've set up `mesheryctl` locally, and would like to take this up

**Comment by Jougan-0** [comment]
This is already assigned to @Redarcher9 you can take up other issue.

**Comment by sudhanshutech** [comment]
@Redarcher9 any progress?

**Comment by Redarcher9** [comment]
Hi @sudhanshutech , I've provided updates to Lee and Yash over slack. I'm little confused over what the next steps could be. I'll tag you in the slack post for further discussion.

**Comment by Redarcher9** [comment]
@Ashparshp  I had provided the update to Lee and Yash over Slack, at that time I was told that this issue would get resolved as a part of other fix. > @Redarcher9, What's the progress?

**Comment by pranjalg1331** [comment]
Whats the progress on this issue? Facing the same error.

**Comment by lekaf974** [comment]
@pranjalg1331 did you fix this error ?

**Comment by pranjalg1331** [comment]
i will start working on it today

**Comment by pranjalg1331** [comment]
please review @lekaf974 @Jougan-0

**Comment by pranjalg1331** [comment]
@Jougan-0 pr only has error handling code now.

**Comment by lekaf974** [comment]
@Jougan-0 could we create a separate issue related to orgId changes so we can merge the PR which fix the panic ?

**Comment by JianMinTang** [comment]
Hi @Jougan-0 and @leecalcote, I think I’ve identified the reason for the panic error when using the create command. It seems that each user has a different organization ID, which you can find in the [URL](http://localhost:9081/api/identity/orgs) after running ./meshery system login. To create a new environment, you’ll need to use your own orgId. Hope this helps!

**Comment by leecalcote** [comment]
Thank you, @JianMinTang 👏 Let's get some maintainer 👀 on your PR and some feedback going. This is good stuff.

**Comment by pranjalg1331** [comment]
@leecalcote my pr #12332 to handle the panic err has not been merged yet and pr #12384 does not seems to resolve it.
